terraform/warehouse/vcl/main.vcl

```diff
@@ -477,6 +477,43 @@ sub vcl_deliver {
     set resp.http.X-Content-Type-Options = "nosniff";
     set resp.http.X-Permitted-Cross-Domain-Policies = "none";
 
+    # See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Permissions-Policy
+    set resp.http.Permissions-Policy = {"
+        publickey-credentials-create=(self),
+        publickey-credentials-get=(self),
+        accelerometer=(),
+        ambient-light-sensor=(),
+        autoplay=(),
+        battery=(),
+        camera=(),
+        display-capture=(),
+        document-domain=(),
+        encrypted-media=(),
+        execution-while-not-rendered=(),
+        execution-while-out-of-viewport=(),
+        fullscreen=(),
+        gamepad=(),
+        geolocation=(),
+        gyroscope=(),
+        hid=(),
+        identity-credentials-get=(),
+        idle-detection=(),
+        local-fonts=(),
+        magnetometer=(),
+        microphone=(),
+        midi=(),
+        otp-credentials=(),
+        payment=(),
+        picture-in-picture=(),
+        screen-wake-lock=(),
+        serial=(),
+        speaker-selection=(),
```

Comment:
There is at least one page on pypi.org that plays audio, would this prevent that?

Reply:
If we're thinking about a page that embeds a player that plays both video and audio, I don't think this setting matters - it's if the underlying JavaScript would **prompt** the user for selecting an audio output device. 
https://w3c.github.io/mediacapture-output/#dom-mediadevices-selectaudiooutput

Is there a particular page you have in mind?

Reply:
Yeah, that's the page. Seems right to me based on those docs!